pallets/services/src/mock.rs

```diff
@@ -232,11 +232,7 @@ impl EvmAddressMapping<AccountId> for PalletEVMAddressMapping {
 	}
 
 	fn into_address(account_id: AccountId) -> H160 {
-		account_id.using_encoded(|b| {
-			let mut addr = [0u8; 20];
-			addr.copy_from_slice(&b[0..20]);
-			H160(addr)
-		})
+		H160::from_slice(&AsRef::<[u8; 32]>::as_ref(&account_id)[0..20])
```

Comment:
just to be safe we should use the same method as in runtime, so that we can be sure

Reply:
This one is tricky to read from the runtime directly, but this is the same implementation of `EnsureAccountIdTruncated` that is used for Withdrawal Origin in the runtime.